cmd/minishift/cmd/root.go

```diff
@@ -179,3 +199,34 @@ func ensureConfigFileExists(configPath string) {
 		}
 	}
 }
+
+func performPostUpdateExecution() error {
+	fmt.Println("Binary has been recently updated.")
```

Comment:
we can formulate this a bit better I think. "Minishift was upgraded from xxx to yyy. Running post update actions" (we can write the previous version into the update file, then we can literally write this)